tests/phpunit/api/v3/ReportTemplateTest.php

```diff
@@ -24,6 +24,8 @@ class api_v3_ReportTemplateTest extends CiviUnitTestCase {
 
   protected $contactIDs = [];
 
+  protected $aclGroupId = NULL;
```

Comment:
Non blocking but I have been changing the report variables to be like `aclGroupID` - as `ID` is an abbreviation & differs from the id that freud so loved.

Reply:
ok fixed